align/schema/pdk.py

```diff
@@ -0,0 +1,119 @@
+from typing import List, Union, Dict, Optional, Tuple
+from align.schema import schema
+from pydantic import validator, ValidationError, Field
+
+
+class ParasiticValues(schema.BaseModel):
+    mean: int = 0
+    min: int = 0
+    max: int = 0
+
+
+class Layer(schema.BaseModel):
+    name: str
+    gds_layer_number: int
+    gds_data_type: Optional[Dict[str, int]] = Field(default_factory=lambda: {"draw": 0})
```

Comment:
We may want to do a more generic gds_info dict instead of gds_layer_number and gds_data_type. 

Would defer this to a separate PR as keeping existing names helps us get this PR merged quickly.